app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -274,12 +277,18 @@ def ridp_agreement
     else
       set_consumer_bookmark_url
     end
+    respond_to do |format|
+      format.html
+    end
   end
 
   def upload_ridp_document
     set_consumer_bookmark_url
     set_current_person
     @person.consumer_role.move_identity_documents_to_outstanding
+    respond_to do |format|
+      format.html
+    end
```

Comment:
I believe if there's not a specific target for the `render` action, this respond_to block can be written as 
```
respond_to :html
```